CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ All notable changes to this project will be documented in this file.
 ### Added
 
 - The associated configuration is now logged for each issued secret ([#413]).
+- Active Directory's `samAccountName` generation can now be customized ([#454]).
```

Comment:
Watch out, this needs to be moved to Unreleased after merge of main

Reply:
https://github.com/stackabletech/secret-operator/pull/454/commits/29eae2c715602026eab06e64512cd1270c865e16